Unity-Technologies/ScriptableRenderPipeline
# Remove call to editor-only XRGraphicsConfig.stereoRenderingMode

### Purpose of this PR
Solves an issue introduced by PR #1933 where build would fail due to access to an editor-only method. 
The check that does this access isn't strictly required- since SRP will only support single-pass double-wide for now, we can assume that if stereo is enabled, we are always rendering single-pass double-wide.

---
### Release Notes
The reason the stereoRenderingMode method is Editor-only is because, for now, the StereoRenderingPath object is only defined in UnityEditor. 

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders/proj57-All%20Tests?ScriptableRenderLoop_branch=xrgc_protection&unity_branch=trunk&automation-tools_branch=add-platform-filter

**Manual Tests**: What did you do?
Manually kicked off a build for Windows x64 (successful)

**Automated Tests**: What did you setup?
None- perhaps we should have builds be part of the SRP test suite?

Any test projects to go with this to help reviewers?
Should work on any empty project.

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low- one-line deletion in HDRP, plus a slightly different conditional in XRGraphicsConfig itself.

**Halo Effect**: None, Low, Medium, High?
Low- XRGraphicsConfig is the interface with XR for SRP, but the changes are to only a currently unused method.

---
### Comments to reviewers
Could do with a test run from @FrancescoC-unity since I don't happen to have an id@xboxone handy.
